CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# dbt_twitter_source v0.4.1
```

Comment:
```suggestion
# dbt_twitter_source v0.4.1
## Fixes
```
In addition to the above header, would you be able to add the models which had this update applied. 

Additionally, be sure to add yourself as a contributor and link this PR! 🎉  You can use [this CHANGELOG entry](https://github.com/fivetran/dbt_fivetran_log/blob/main/CHANGELOG.md#dbt_fivetran_log-v062) as a reference.

Reply:
Applied the suggestions in [086710c](https://github.com/fivetran/dbt_twitter_source/pull/15/commits/086710ca32e3bd77ad126dee9846e58304b7fe61)